collections/sentinel-3-l1b-slstr.yaml

```diff
@@ -31,7 +31,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-3-slstr-l1b/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-3-slstr-l1b/#attribution-and-use" target="_blank">Attribution and Use </a>.
```

Comment:
Attribution and Use > Sentinel-3 SLSTR Attribution and Use

Reply:
done